pom.xml

```diff
@@ -312,8 +313,8 @@
                 </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maven-deploy-plugin</artifactId>
-                    <version>${maven-deploy.version}</version>
+                    <artifactId>maven-clean-plugin</artifactId>
+                    <version>3.0.0</version>
```

Comment:
all of these plugin changes should also be using `maven-<whatever>.version` variables.